platforms/esp32/main/main.c

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2024 Husarnet sp. z o.o.
+// Authors: listed in project_root/README.md
+// License: specified in project_root/LICENSE.txt
+
```

Comment:
Add a big-ass banner here, saying that this is just an example and does not have to be built into all of the projects. (Normally I'd say that renaming would be the best option, but I doubt it's possible here)

Reply:
Renamed to a `test-runner.c` as this project will be used for integration tests in the future